src/ed.cpp

```diff
@@ -405,10 +359,11 @@ bool srvSimpleQuery(ed_msgs::SimpleQuery::Request& req, ed_msgs::SimpleQuery::Re
     geo::Vector3 center_point;
     geo::convert(req.center_point, center_point);
 
-    for(ed::WorldModel::const_iterator it = ed_wm->world_model()->begin(); it != ed_wm->world_model()->end(); ++it)
-    {
-//        std::cout << it->first << std::endl;
+    // Make a copy of the WM, to keep it thead safe
+    ed::WorldModel wm = *ed_wm->world_model();
 
+    for(ed::WorldModel::const_iterator it = wm.begin(); it != wm.end(); ++it)
```

Comment:
Since line 363 is intended for this loop, I'd remove line 364 here

Reply:
Done